File: src/2845/main.js
var n = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n').map(function(e) { return e.split(' ').map(function(e) { return parseInt(e); }); });
var x = n[0][0] * n[0][1];
console.log(n[1].map(function(e) {
  return e - x;
}).join(' '));

File: src/2845/main.rs
use std::io;

fn main() {
    let mut s = String::new();

    io::stdin().read_line(&mut s).unwrap();

    let values:Vec<i32> = s
        .as_mut_str()
        .split_whitespace()
        .map(|s| s.parse().unwrap())
        .collect();

		let l = values[0];
		let p = values[1];

		let x = l * p;

		s.clear();
    io::stdin().read_line(&mut s).unwrap();

    let values:Vec<i32> = s
        .as_mut_str()
        .split_whitespace()
        .map(|s| s.parse::<i32>().unwrap())
				.map(|s| s - x)
        .collect();

		println!("{}", values.into_iter().map(|x| x.to_string()).collect::<Vec<String>>().join(" "));
}
